### PR TITLE
search: lint internal/query and search_results.go

### PR DESCRIPTION
**Pull request by rvantonder**
Stacked on #18262.

lint

### CODE REVIEW

File: cmd/frontend/graphqlbackend/search_results.go [hunk 1]
```diff
@@ -1709,7 +1709,7 @@ func (r *searchResolver) doResults(ctx context.Context, forceOnlyResultType stri
 		wg.Add(1)
 		goroutine.Go(func() {
 			defer wg.Done()
-			agg.doFilePathSearch(ctx, &argsIndexed)
+			_ = agg.doFilePathSearch(ctx, &argsIndexed)
```
Comment (rvantonder) [inline cmd/frontend/graphqlbackend/search_results.go:1712]: I think this will silence the golint.

Comment (camdencheek) [reply cmd/frontend/graphqlbackend/search_results.go:1712]: Nice, this was bothering me

Comment (rvantonder) [reply cmd/frontend/graphqlbackend/search_results.go:1712]: looks like it's working. pls to stamp?

Comment (camdencheek) [reply cmd/frontend/graphqlbackend/search_results.go:1712]: ope. Thought I did again

**Review by camdencheek — APPROVED** [review]

**Pull request merged**